Modules/EMCAL/include/EMCAL/CellTask.h

```diff
@@ -155,6 +148,23 @@ class CellTask final : public TaskInterface
   o2::emcal::BadChannelMap* mBadChannelMap;                  ///< EMCAL channel map
   o2::emcal::TimeCalibrationParams* mTimeCalib;              ///< EMCAL time calib
   int mTimeFramesPerCycles = 0;                              ///< TF per cycles
+
+  TH1* mEvCounterTF = nullptr;      ///< Number of Events per timeframe
+  TH1* mEvCounterTFPHYS = nullptr;  ///< Number of Events per timeframe per PHYS
+  TH1* mEvCounterTFCALIB = nullptr; ///< Number of Events per timeframe per CALIB
+  TH1* mTFPerCyclesTOT = nullptr;   ///< Number of Time Frame per cycles TOT
+  TH1* mTFPerCycles = nullptr;      ///< Number of Time Frame per cycles per MonitorData
+  TH1* mCellsMaxSM = nullptr;       ///< Supermodule with the largest amount of cells
+  double mCellThreshold = 0;        //
```

Comment:
Should get a doxygen documentation (starting with `///<`), but can be done in the next iteration.